src/NServiceBus.Core.Tests/DelayedDelivery/TimeoutManager/TimeoutRecoverabilityBehaviorTest.cs

```diff
@@ -1,57 +0,0 @@
-﻿namespace NServiceBus.Core.Tests.Timeout.TimeoutManager
-{
-    using System;
-    using System.Collections.Generic;
-    using System.IO;
-    using System.Threading.Tasks;
-    using NServiceBus.Pipeline;
-    using NServiceBus.Transports;
-    using NUnit.Framework;
-
-    public class TimeoutRecoverabilityBehaviorTest
-    {
-        [Test]
-        public async Task Invoke_when_timeout_fails_should_invoke_bus_notifications()
```

Comment:
I'd say this is already covered by acceptance tests:

https://github.com/Particular/NServiceBus/blob/remove-notifications-from-tm-recoverability/src/NServiceBus.AcceptanceTests/DelayedDelivery/TimeoutManager/When_timeout_dispatch_fails.cs#L33

and here

https://github.com/Particular/NServiceBus/blob/remove-notifications-from-tm-recoverability/src/NServiceBus.AcceptanceTests/DelayedDelivery/TimeoutManager/When_timeout_storage_fails.cs#L35

This specific unit test was created solely for the purpose of testing notifications. What's the value in keeping it?